### ISSUE TITLE
"German" label is not defined for the GIFs when the language is set in german.

### ISSUE THREAD

**Issue by msftedad** [Giphy/GiphyAPI#253]
**Description:**
After changing the Teams language to German, screen reader is announcing the GIFs names in English only.

**Test Environment:**
OS build:  22H2 OS Build 22621.1702
Ring Version: Public.
Team’s version: 1.6.00.12455 (64-bit) 
Screen reader: JAWS version: 2023.2212.23 ILM, NVDA 2023

**Pre-Requisites**
Change the Teams language to German(Settings and more-Settings-General tab-Language).

**Repro Steps**
1.Open Teams app.
2.Navigate to "Chat" on the app bar and activate by using Enter key.
3.Select any chat.
4.Navigate to "GIFs" button and activate by using Enter key.
5.Navigate on the GIFs and observe the screen reader announcement.

**Actual Results**
Screen reader is announcing English name of the GIFs when the language of Teams is set to German.

**Expected Results**
German name should also defined for the GIFs and screen reader should announce the German names of the GIFs when Teams language is set German.

**Additional notes**
Issue is repro in public build
https://giphy.com/

https://github.com/Giphy/GiphyAPI/assets/93735775/577aaf61-5e5a-43c5-baf0-61d7d2d2b2ed

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?
